source/core/initializable.ts

```diff
@@ -77,7 +77,7 @@ export function uninitialize() {
 
 /**
  * Method decorator for asserting the initialization status of an initializable to be true.
- * @see { @link Initializable.assertInitialized }
+ * @see {@link Initializable.assertInitialized }
```

Comment:
Minor: the link doesn't show up in the generated docs, there's just a box with `see`.